systemtest/src/main/java/io/strimzi/systemtest/utils/specific/BridgeUtils.java

```diff
@@ -4,192 +4,76 @@
  */
 package io.strimzi.systemtest.utils.specific;
 
-import io.netty.handler.codec.http.HttpResponseStatus;
-import io.strimzi.systemtest.Constants;
 import io.strimzi.systemtest.utils.HttpUtils;
 import io.strimzi.test.TestUtils;
-import io.vertx.core.MultiMap;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-import io.vertx.ext.web.client.HttpResponse;
-import io.vertx.ext.web.client.WebClient;
-import io.vertx.ext.web.codec.BodyCodec;
+import io.vertx.core.http.HttpMethod;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 import java.io.InputStream;
-import java.util.Collections;
 import java.util.Map;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static io.strimzi.systemtest.resources.ResourceManager.cmdKubeClient;
 
 public class BridgeUtils {
 
     private static final Logger LOGGER = LogManager.getLogger(HttpUtils.class);
 
     private BridgeUtils() { }
 
-    public static JsonObject generateHttpMessages(int messageCount) {
-        LOGGER.info("Creating {} records for KafkaBridge", messageCount);
-        JsonArray records = new JsonArray();
-        JsonObject json = new JsonObject();
-        for (int i = 0; i < messageCount; i++) {
-            json.put("value", "msg_" + i);
-            records.add(json);
+    public static String getCurlCommand(HttpMethod httpMethod, String url, String headers, String data) {
```

Comment:
I would prefer called it `buildCurlCommand` but it's just  suggestion.

Reply:
I have another suggestion, but it's just that. You could also check that data has to be empty when the method is not  PUT or POST.